src/util/singleton.h

```diff
@@ -14,15 +14,16 @@ class Singleton {
     }
 
     static T* instance() {
-        if (m_instance == NULL) {
-            qWarning() << "Singleton class has not been created yet, returning NULL";
+        if (m_instance == nullptr) {
```

Comment:
`VERIFY_OR_DEBUG_ASSERT(m_instance)`? A developer should better notice whenever this happens. Not related to this PR, but why not improve the code if possible.